Automattic/woocommerce-payments
# Disabling the manual refunds button unless a WCPay refund has failed.

Fixes #426

#### Changes proposed in this Pull Request

- Save the refund status in `_wcpay_refund_status` as a meta value.
- Use the meta value to conditionally disable the manual refund button.

Context: Payments made through WooCommerce Payments must be refunded through WCPay. Only when the refund fails, the funds may be refunded manually.

#### Testing instructions

1. Make a purchase with test card `4000 0000 0000 5126`.
2. Navigate to the order screen and click "Refund" to display the refund bar.
3. Make sure that the button for manual refunds is disabled and explains that "Refunds are available only through WooCommerce Payments."
4. Attempt a refund through WCPay.
5. When the refund fails and the page is reloaded, the manual payments button should be enabled, but the WCPay one missing.
